engine/src/test/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActorSpec.scala

```diff
@@ -59,7 +63,7 @@ class WorkflowExecutionActorSpec extends CromwellTestkitSpec with BeforeAndAfter
       val isFatal = (_: Throwable) => false
 
       // Sleep a bit, and let the metadata do its thing in the background.
-      Thread.sleep(1000L)
+      Thread.sleep(3.seconds.dilated.toMillis)
```

Comment:
We'd still need the `.toMillis` to convert the value to a `java.lang.Long`.

I weep with you on a lot of these tests that fail intermittently. Developer's choice?